src/Ocelot/Configuration/Creator/DefaultMetadataCreator.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Ocelot.Configuration.Builder;
+using Ocelot.Configuration.File;
+
+namespace Ocelot.Configuration.Creator;
```

Comment:
Wrong namespace and folder!
I guess this class should be located in the Metadata folder and it should be defined in `Ocelot.Metadata` namespace.

Reply:
why, sorry, I'm following the current logic, the creators are in the creator folder.